Relax requirements for qcelemental as older versions are permitted for older Python versions.
From qcelemental pyproject.toml:
"qcel is compatible with most any numpy, v1 or v2, but numpy v2 only works with pint >=0.24, which is only available for py >=3.10"
neil.douglas@york.ac.uk
--- pyproject.toml.orig	2025-05-01 11:36:57.511605888 +0100
+++ pyproject.toml	2025-05-01 11:38:27.427595199 +0100
@@ -28,15 +28,15 @@
 [tool.poetry.dependencies]
 numpy = [
     { version = ">=1.12.0", python = "3.8" },
-    { version = ">=1.26.1,<2.0", python = ">=3.9,<3.10" },
-    { version = ">=1.26.1", python = ">=3.9,<3.13" },
+    { version = ">=1.12.0,<2.0", python = ">=3.9,<3.10" },
+    { version = ">=1.12.0", python = ">=3.9,<3.13" },
 ]
 # qcel is compatible with most any numpy, v1 or v2, but numpy v2 only works with pint >=0.24, which is only available for py >=3.10
 python = "^3.7"
 pint = [
     { version = ">=0.10", python = ">=3.7,<3.9" },
-    { version = ">=0.23", python = ">=3.9,<3.10" },
-    { version = ">=0.24", python = ">=3.10,<3.13" },
+    { version = ">=0.10", python = ">=3.9,<3.10" },
+    { version = ">=0.10", python = ">=3.10,<3.13" },
 ]
 pydantic = ">=1.8.2"
 nglview = { version = "^3.0.3", optional = true }
